### PR TITLE
Added hang glider readme, fixed hang glider music, documented platformer main.fab.

### PR DESCRIPTION
**Pull request by msbeck**
Readme is now provided for the hang glider example. Music.txt error when re-importing resolved.

### CODE REVIEW

File: examples/hang_glider/music.txt
Comment (pubby) [inline examples/hang_glider/music.txt:1]: I think it should be `-1` instead of `1`

**Comment by pubby** [comment]
Thanks for doing this! Looks good, so whenever you're ready and happy with it, remove the 'draft status' and I can commit it.

**Comment by msbeck** [comment]
Fully commented main.fab for platformer also now included in this pull request.

**Comment by pubby** [comment]
Thanks!

**Pull request merged**